DAQ/src/CaloHitsFromFragments_module.cc

```diff
@@ -357,8 +368,10 @@ void art::CaloHitsFromFragments::analyze_calorimeter_(
       //      }
       float time = hits[hitIdx].first.Time + peakIndex * digiSampling_ + timeCalib_[sipmID];
 
+      bool  isCaphri = std::find(caphriCrystalID_.begin(), caphriCrystalID_.end(), crystalID) != caphriCrystalID_.end();
       // FIX ME! WE NEED TO CHECK IF TEH PULSE IS SATURATED HERE
-      if (eDep < hitEDepMax_) {
+      if ( ( (eDep >= hitEDepMin_) || (isCaphri && (eDep >= caphriEDepMin_))) &&
+           ( (eDep <  hitEDepMax_) || (isCaphri && (eDep <  caphriEDepMax_))) ) {
```

Comment:
The first line looks like it expects caphriEDepMin < histEDepMin ( ie a lower threshold for caphri hits).  Will this logic do the right thing if caphriEDepMin > histEDepMin.  If that's an illegal configuration, then protect against it in the c'tor.  A similar comment for the second line.

Reply:
I understand your point. We are still in a developing stage for the proper CAPHRI simulation/configuration, so I think we can revisit it once that will be more clear. During the summer time we should have collaborators optimizing this part of the code and this change is intended to help the process.

Reply:
You can address this in a later PR.  Please be sure that, as this work evolves, the c'tor has tests for logical consistency of the configuration. Then you can assume consistency in the rest of the code.

And thanks for fixing the typo.